README.rst

```diff
@@ -4,53 +4,114 @@ TruFont
 =======
 
 `TruFont <https://trufont.github.io>`__ is a font-editing application
-written with Python3, ufoLib, defcon and PyQt5.
+written with Python3, ufoLib, defcon, and PyQt5.
+
+.. image:: Lib/trufont/resources/screenshot.png
 
 Getting started
-~~~~~~~~~~~~~~~
+===============
+
+Quick installation instructions
+-------------------------------
+
+If you have Python3 and Git installed and want to quickly install 
+and run TruFont in a virtual environment, open a terminal and enter 
+the following chain of commands as one line:
+
+``python3 -m venv trufont-venv && source trufont-venv/bin/activate && cd trufont-venv && git clone https://github.com/trufont/trufont.git && cd trufont && pip install --upgrade -r requirements.txt && pip install --editable . && trufont``
+
+Or, as separate lines, making modifications when needed:
+
+   .. code::
+
+     python3 -m venv trufont-venv
+     source trufont-venv/bin/activate
+     cd trufont-venv
+     git clone https://github.com/trufont/trufont.git
+     cd trufont
+     pip install --upgrade -r requirements.txt
+     pip install --editable .
+     trufont
+
+Detailed Installation Instructions
```

Comment:
Maybe rename this Contributor Installation Instructions and include the above 'all on 1 line' and 'all at once' commands, then go on to explain them?